includes/template-library/manager.php

```diff
@@ -227,9 +227,10 @@ public function get_library_data( array $args ) {
 		Plugin::$instance->documents->get_document_types();
 
 		$filter_sources = ! empty( $args['filter_sources'] ) ? $args['filter_sources'] : [];
+		$force_update = ! empty( $args['sync'] );
```

Comment:
Irrelevant to your changes, but I just noticed this in the function:
`if ( empty( $library_data ) ) {
			return $library_data;
		}`
Does it even make sense?

Reply:
Not really but Udi wrote it for fixing a specific bug ([ED-5715] (#19246)), I'm assuming there is a reason for that.

Reply:
Weird, maybe worth checking with him?

Reply:
I checked with him, he doesn't remember.
I prefer to leave it as is, just in case.

Reply:
Ok, cool